cirq/experiments/cross_entropy_benchmarking_test.py

```diff
@@ -66,3 +66,6 @@ def test_cross_entropy_benchmarking():
     assert np.isclose(np.mean(fidelities_1), 1.0, atol=0.1)
     assert np.isclose(np.mean(fidelities_2), 1.0, atol=0.1)
     assert len(fidelities_3) == 1
+
+    # Sanity test that plot does not barf.
```

Comment:
What's barfing referring to?

Reply:
Just making sure it runs.  Updated comment.